Fix typo in persistent caching (#7)
webpack/changelog-v5: guides/persistent-caching.md

```

This invalidates the persistent cache when anything in `webpack/lib` or in dependencies of webpack like `watchpack`, `enhanced-resolved`, etc. changes.

> Coincidentally as this always a default value, so you don't have to specify it.

Reviewer: ```suggestion
Coincidentally this is already the default value, so you don't have to specify it.
```
